src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/controller/RecommendationController.kt

```diff
@@ -82,13 +82,29 @@ internal class RecommendationController(
   suspend fun generatePartADocument(
     @PathVariable("recommendationId") recommendationId: Long,
     @RequestBody createPartARequest: CreatePartARequest,
-  ): ResponseEntity<PartAResponse> {
+  ): ResponseEntity<DocumentResponse> {
     log.info(normalizeSpace("Generate Part A document endpoint for recommendation id: $recommendationId"))
 
     val responseEntity = try {
       ResponseEntity(recommendationService.generatePartA(recommendationId, authenticationFacade.currentNameOfUser, createPartARequest.userEmail), OK)
     } catch (e: UserAccessException) {
-      ResponseEntity(PartAResponse(Gson().fromJson(e.message, UserAccessResponse::class.java)), FORBIDDEN)
+      ResponseEntity(DocumentResponse(Gson().fromJson(e.message, UserAccessResponse::class.java)), FORBIDDEN)
+    }
+    return responseEntity
+  }
+
+  @PreAuthorize("hasRole('ROLE_MAKE_RECALL_DECISION')")
+  @PostMapping("/recommendations/{recommendationId}/task-list-no-recall")
```

Comment:
Just wondering if the text 'task-list' is right here? We don't have it in the Part A endpoint name so thinking we should make it consistent? Maybe add 'task-list' to the Part A endpoint or remove from his one?